EnglishFilter/sections/specific.txt

```diff
@@ -262,6 +262,8 @@ fileriver.net##center > a[onclick*="https://href.li"]
 fileriver.net##.wlm-container
 britannica.com##.video-ad-container
 familyporn.tv##.thumbs > div.side_spot
+||jumboporn.xyz/images/eva.gif
+||jumboporn.xyz/t63fd79f7055.js
```

Comment:
I don't see it with US IP.

Reply:
RU IPs.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/102485205-55858900-4078-11eb-8651-a114cd457a12.png)</details><br/>